amivapi/studydocs/endpoints.py

```diff
@@ -80,6 +81,11 @@
                 'maxlength': 30,
                 'nullable': True,
                 'type': 'string'
+            },
+            'coursesemester': {
+                    'maxlength': 5,
+                    'nullable': True,
+                    'type': 'string'
```

Comment:
Ok, I see your point. But then we maybe should use 'courseyear' or something and validate its a year (e.g. with regex) - I think this is important to ensure that this field is comparable accross several items.

E.g. the first person could write: FS12, the next person 2014, etc - this makes it very complicated for a tool (e.g. the website) to parse it. 

I have created an issue for this: #136